src/edu/csus/ecs/pc2/core/packet/PacketFactory.java

```diff
@@ -357,6 +357,11 @@ public final class PacketFactory {
 
     public static final String REQUEST_LOGIN_AS_PROXY = "REQUEST_LOGIN_AS_PROXY";
     
+    /**
+     * Ultimate destination on where answers to clarification is sent to
```

Comment:
If these are groups, it should say that.

Reply:
removed the comments.